Firmware/variants/MK25-RAMBo10a.h

```diff
@@ -265,6 +265,7 @@
  *------------------------------------*/
 
 //#define HOST_SHUTDOWN              //Host supports "//action:shutdown" feature
+//#define REPLACE_SETREADY           //Replace "Set Ready"/"Set not Ready" menu entry with "Print from host" menu entry
```

Comment:
I feel the description should say more what this option does. For example to answer the question "Why/When would I want to replace set ready?"

The description here is likely our only documentation :)

Reply:
I also updated the description for the shutdown feature.